packages/rocketchat-mapview/client/mapview.coffee

```diff
@@ -10,15 +10,17 @@ class MapView
 		mv_enabled = RocketChat.settings.get 'MapView_Enabled'
 		mv_googlekey = RocketChat.settings.get 'MapView_GMapsAPIKey'
 
-		if message.location and mv_enabled
+		if message.location
 
 			# GeoJSON is reversed - ie. [lng, lat]
 			longitude = message.location.coordinates[0]
 			latitude = message.location.coordinates[1]
 
 			# confirm we have an api key set, and generate the html required for the mapview
-			if mv_googlekey?.length
+			if mv_enabled and mv_googlekey?.length
```

Comment:
Can we keep the old if here? If admin disable the MapView but keep the GMaps Api Key we can keep rendering the map image.